src/ape_test/provider.py

```diff
@@ -348,7 +348,7 @@ def get_virtual_machine_error(self, exception: Exception, **kwargs) -> VirtualMa
             else:
                 return VirtualMachineError(base_err=exception, **kwargs)
 
-        elif isinstance(exception, ContractPanicError):
```

Comment:
reviewer note: `ContractPanicError` is a subclass of web3's `ContractLogicError`